packages/react-components/src/organisms/__tests__/ToolModal.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import ToolModal from '../ToolModal';
+
+const props: ComponentProps<typeof ToolModal> = {
+  backHref: '/wrong',
+  title: '',
+};
+it('renders the title', () => {
+  const { getByText } = render(<ToolModal {...props} title="ModalTitle" />);
+  expect(getByText('ModalTitle', { selector: 'h3' })).toBeVisible();
+});
+
+it('renders default values into inputs', () => {
+  const { queryAllByRole } = render(
+    <ToolModal
+      {...props}
+      name="LinkName"
+      description="LinkDescription"
+      url="http://example.com"
+    />,
+  );
+  expect(queryAllByRole('textbox').map((input) => input.getAttribute('value')))
+    .toMatchInlineSnapshot(`
+    Array [
+      "http://example.com",
+      "LinkName",
+      "LinkDescription",
+    ]
+  `);
+});
+
+it('triggers the save function', () => {
```

Comment:
Check data as well

Reply:
Going to do it in the next PR. Saving might change